src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    const [key, value] = style.split(':').map((item) => item.trim());
+
+    if (key && value) {
+      stylesObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to build the `stylesObject`, consider using array methods like `reduce` which is more suitable for transforming an array into an object.